### PR TITLE
fix: contained tokenlist height

### PR DESCRIPTION
**Pull request by just-toby**
fixes a bug where the token list was being cut off in the non-page-centered version of the widget.

### CODE REVIEW

File: src/components/TokenSelect/index.tsx [hunk 1]
```diff
@@ -98,7 +102,12 @@ export function TokenSelectDialogContent({ value, onSelect, onClose }: TokenSele
     )
   }
   return (
-    <TokenSelectContainer>
+    <TokenSelectContainer
+      style={{
+        minWidth: isPageCentered ? Math.min(400, width) : 'auto',
+        minHeight: isPageCentered ? 'unset' : '100%',
+      }}
```
Comment (JFrankfurt) [inline src/components/TokenSelect/index.tsx:109]: can you do this with `$isPageCentered` as a transient styled-component prop and something like `min(400px, 100vw)`?

**Review by JFrankfurt — CHANGES_REQUESTED** [review]
see comment

File: src/components/TokenSelect/index.tsx (hunk 1 above)
Comment (just-toby) [reply src/components/TokenSelect/index.tsx]: yep

**Comment by just-toby** [comment]
https://user-images.githubusercontent.com/66155195/222813938-9665811f-3722-4120-a2c2-68dd86833c37.mov

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**